files/scripts/syncd.sh

```diff
@@ -46,6 +46,10 @@ function startplatform() {
 function waitplatform() {
     
     if [[ x"$sonic_asic_platform" == x"mellanox" ]]; then
+        BOOT_TYPE=`getBootType`
+        if [[ x"$BOOT_TYPE" == x"fast" ]]; then
+            return
```

Comment:
It would be meaningful to have a log line here mentioning that for fastboot (or warm), `pmon` start will be delayed as governed by `files/build_templates/pmon.timer`

Reply:
Log added.